tests/trainer/flags/test_fast_dev_run.py

```diff
@@ -19,3 +24,75 @@ def test_skip_on_fast_dev_run_tuner(tmpdir, tuner_alg):
     expected_message = f'Skipping {tuner_alg} since fast_dev_run is enabled.'
     with pytest.warns(UserWarning, match=expected_message):
         trainer.tune(model)
+
+
+@pytest.mark.parametrize('fast_dev_run', [1, 4])
+@mock.patch.dict(os.environ, {"PL_DEV_DEBUG": "1"})
+def test_callbacks_and_logger_not_called_with_fastdevrun(tmpdir, fast_dev_run):
```

Comment:
is it just move from other file or can you pls tell us what is the diff?

Reply:
not exactly. Before this PR, `ModelCheckpoint` was enabled with `fast_dev_run`, but now it isn't. Also updated the test to assert everything that is required to be disabled (callbacks and loggers) so thought it's better to keep this one along with other `fast_dev_run` tests.